packages/react-resizable-panels/package.json

```diff
@@ -81,8 +81,8 @@
     "react-dom": "experimental"
   },
   "peerDependencies": {
-    "react": "^16.14.0 || ^17.0.0 || ^18.0.0",
-    "react-dom": "^16.14.0 || ^17.0.0 || ^18.0.0"
+    "react": "^16.14.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^19.0.0-rc",
+    "react-dom": "^16.14.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^19.0.0-rc"
```

Comment:
I _think_ the `^19.0.0-rc` bit may not be necessary. Is it?

Reply:
If you want to support NextJS 15 which currently installs the RC, then it is necessary

You can test: https://semver.npmjs.com/

<img width="515" alt="Screenshot 2024-10-22 at 15 58 40" src="https://github.com/user-attachments/assets/22385b44-0e63-4142-9d05-b35737d00e9f">

Reply:
That's interesting. I was testing using [this tool](https://jubianchi.github.io/semver-check/#/^19.0.0/19.0.0-rc-123) which seemed to suggest that it was unnecessary. I guess it's safer to use the more conservative approach though

Reply:
@bvaughn  The other data point is radix-ui using the same, see:
https://github.com/radix-ui/primitives/blob/main/packages/react/dialog/package.json#L49

I am not sure about the way the package managers handle this, but generally 19.0.0 is considered higher than 19.0.0-rc.x, so if you specify "^19.0.0", you wouldn't want 19.0.0-rc (which was released before 19.0.0) to be accepted